README.md

```diff
@@ -43,13 +43,9 @@ Minimal Example `flake.nix`:
 {
   inputs.dream2nix.url = "github:nix-community/dream2nix";
   outputs = { self, dream2nix }@inputs:
-    let
-      dream2nix = inputs.dream2nix.lib.init {
-        # modify according to your supported systems
-        systems = [ "x86_64-linux" ];
-        config.projectRoot = ./. ;
-      };
-    in dream2nix.makeFlakeOutputs {
+    dream2nix.lib.makeFlakeOutputs {
+      systems = ["x86_64-linux"];
+      # config.projectRoot defaults to source
```

Comment:
Yeah got it. I was thinking it would be convenient for most repos, but yeah it is probably best to explicitly set that so users don't get confused later on.